rflx/pyrflx/typevalue.py

```diff
@@ -562,6 +562,7 @@ def __init__(
             self._last_field = INITIAL.name
         else:
             self._preset_fields(INITIAL.name)
+        self.message_last_name = Last("Message")
```

Comment:
`message_last_name` extends the public interface of `MessageValue` while its value is constant and only used internally. It would be better to use `Last("Message")` directly, create a module local constant (e.g., `MESSAGE_LAST`) or at least to make the attribute private.

Reply:
Fixed in https://github.com/Componolit/RecordFlux/pull/424/commits/98ca5ff4afc1ed3ecea774e373e6a621a7dfee38.